### ISSUE TITLE
read.argo() perhaps should read MTIME

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#2063]
See https://github.com/ArgoCanada/argoFloats/issues/590, where discussion ought to continue, rather than here. (Split discussions are hard to follow.)

**Comment by richardsc** [comment]
From ArgoCanada/argoFloats#590:

> One thought though, as an issue for oce, is that perhaps as.ctd() on an argo object should know how to interpret time (metadata) and mtime to be converted to the standard fields (for a ctd object) of startTime (metadata) and time (data).

**Comment by dankelley** [comment]
Oh, it reads it already.  I was being silly.  I'm making a new issue for `as.ctd()`.
